src/openzeppelin/utils/constants.cairo

```diff
@@ -36,3 +36,43 @@ const IACCESSCONTROL_ID: u32 = 0x7965db0b_u32;
 //
 
 const DEFAULT_ADMIN_ROLE: felt252 = 0;
+
+
+//
+// Selectors
+//
+
+const NAME_SELECTOR: felt252 = 0x361458367e696363fbcc70777d07ebbd2394e89fd0adcaf147faccd1d294d60;
+const SYMBOL_SELECTOR: felt252 = 0x216b05c387bab9ac31918a3e61672f4618601f3c598a2f3f2710f37053e1ea4;
+const TOKEN_URI_SELECTOR: felt252 =
+    0x226ad7e84c1fe08eb4c525ed93cccadf9517670341304571e66f7c4f95cbe54;
+const TOKENURI_SELECTOR: felt252 =
+    0x362dec5b8b67ab667ad08e83a2c3ba1db7fdb4ab8dc3a33c057c4fddec8d3de;
+const BALANCE_OF_SELECTOR: felt252 =
+    0x35a73cd311a05d46deda634c5ee045db92f811b4e74bca4437fcb5302b7af33;
+const BALANCEOF_SELECTOR: felt252 =
```

Comment:
I think we're all in agreement with moving these to `selectors.cairo` https://github.com/OpenZeppelin/cairo-contracts/pull/625#discussion_r1245924546